tests/e2e_decorrelation/test_error_cases.py

```
@@ -0,0 +1,715 @@
+        assert_plan_structure(decorrelated_plan, {})
+        results = execute_and_fetch_all(executor, decorrelated_plan)
+        # with pytest.raises(Exception, match="single column"):
+        #     logical_plan = parser.parse(sql)
+        #     bound_plan = binder.bind(logical_plan)
+        #     decorrelated_plan = decorrelator.decorrelate(bound_plan)
+
+
+class TestAmbiguousReferences:
+    """Test ambiguous column reference errors."""
+
+    def test_ambiguous_correlation_column(self, catalog, setup_test_data):
```

Ruff (0.14.5)

Unused method argument: `setup_test_data`

(ARG002)